shared/js/initApp.js

```diff
@@ -240,6 +283,9 @@ if (appOptions.droplet) {
         return deferred;
       });
     }
+  } else if (appOptions.level.projectTemplateLevelName) {
+    // this is an embedded project
+    promise.then(dashboard.loadEmbeddedProject(appOptions.level.projectTemplateLevelName));
```

Comment:
Ah I see what you're saying.  This isn't like standalone projects where failing to load the app data means we just default to a blank slate.  The app data *has* to load or the level won't be playable.